doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_mysql_sink.md

```diff
@@ -227,7 +227,7 @@ If `attr_persistence=colum` then `NGSIMySQLSink` will persist the data within th
 | enable\_grouping | no | false | <i>true</i> or <i>false</i>. Check this [link](./ngsi_grouping_interceptor.md) for more details. ||
 | enable\_name\_mappings | no | false | <i>true</i> or <i>false</i>. Check this [link](./ngsi_name_mappings_interceptor.md) for more details. ||
 | enable\_lowercase | no | false | <i>true</i> or <i>false</i>. |
-| enable\_last\_data | no | false | <i>true</i> or <i>false</i>, to enable last data functionality. Check this [link](./last_data_function.md) for more details. |
+| last\_data\_mode | no | upsert | <i>upsert</i> or <i>insert</i> or <i>both</i>, to set last data mode. Check this [link](./last_data_function.md) for more details. |
```

Comment:
I understand that the configuration parameters has changed, from enable_last_data to last_data_mode

Does it involves some risk from the point of view of backward compability (some Cygnus may be using enable_last_data and now get broken with the change)? But maybe the change is safe if this functionality has not been widely used yet.

Reply:
At this moment last_data is few used.
The changed proposed in this PR is backward compatible. Old `enable_last_data` was false by default, and current `last_data_mode` default is insert, which are equivalent.